src/App.jsx

```diff
@@ -1,27 +1,35 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
 import './App.css'
 import Home from "./pages/home/Home";
 import Blog from "./pages/blog/Blog";
 import Contact from "./pages/contact/Contact";
 
 import Signup from "./pages/signup/Signup";
-import {BrowserRouter as Router, Routes, Route, Link} from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
 import { LoginPage } from './pages/login';
-
+import Logo from "./assets/logo.jpg";
 
 function App() {
   return (
     <>
-    <div className="App">
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="blog" element={<Blog />} />
-        <Route path="contact" element={<Contact />} />
-        <Route path="login" element={<LoginPage />} />
-        <Route path="signup" element={<Signup />} />
-      </Routes>
-    </div>
+      <div className="App">
+        <nav className="nav-bar">
+          <ul>
+            <img src={Logo} alt="logo" />
+            <li><Link to="/" className="links">HOME</Link></li>
+            <li><Link to="blog" className="links">BLOG</Link></li>
+            <li><Link to="contact" className="links">CONTACT</Link></li>
+            <li><Link to="login" className="links">LOGIN</Link></li>
+            <li><Link to="signup" className="links">SIGN UP</Link></li>
```

Comment:
In terms of CSS responsivity for these `<NavBar />` items, when running this branch, it looks like we're having some issues with the menu text on smaller screen sizes. 

I think best practice is hiding the text menu on smaller screen sizes and creating a "hamburger" menu with the NavBar items in some kind of vertical menu?